.github/workflows/pullpreview.yml

```diff
@@ -96,6 +104,10 @@ jobs:
           instance_type: ${{ inputs.INSTANCE_TYPE }}
           # Ports to open on the server
           ports: 80, 443
+          # Use a custom domain for the preview environment
+          dns: ${{ inputs.DNS }}
+          # Maximum length of fully qualified domain name. Cannot be greater than 62 characters.
+          max_domain_length: ${{ inputs.MAX_DOMAIN_LENGTH }}
```

Comment:
add the new keys to the example and comment them out.

Reply:
Done.